src/notifications/pollNotifications.ts

```diff
@@ -32,6 +33,9 @@ export function cancelNotificationsPolling(): void {
 export async function doPollNotifications(
   context: vscode.ExtensionContext
 ): Promise<void> {
+  if (ONPREM) {
```

Comment:
irelevant you do not call pollNotifications

Reply:
it is called on every selection suggestion